commands/solenoid/SolenoidSet.java

```diff
@@ -23,34 +26,33 @@
 	 * @param booleanSuppliers conditions that if true, prevents solenoidSubsystem
 	 *                         from setting
 	 */
-	public SolenoidSet(String name, SolenoidSubsystem system, SolenoidState state,
-			BooleanSupplier... booleanSuppliers) {
-		super(name, system);
+	public SolenoidSet(String name, SolenoidSubsystem system, SolenoidState state, BooleanSupplier... booleanSuppliers) {
 		this.system = system;
 		this.state = state;
 		this.booleanSuppliers = booleanSuppliers;
+
+		requirements = Set.of((Subsystem) system);
 	}
 
 	/**
 	 * Sets the state of a SolenoidSubsystem
 	 * 
-	 * @param system SolenoidSubsystem to set
-	 * @param state  state to set system
+	 * @param name             Name of the Command
+	 * @param system           SolenoidSubsystem to set
+	 * @param state            state to set system
+	 * @param booleanSuppliers conditions that if true, prevents solenoidSubsystem
+	 *                         from setting
 	 */
 	public SolenoidSet(SolenoidSubsystem system, SolenoidState state, BooleanSupplier... booleanSuppliers) {
 		this("SolenoidSet", system, state, booleanSuppliers);
 	}
 
-	public SolenoidSet(String name, SolenoidSubsystem system, SolenoidState state) {
-		this(name, system, state, () -> {
+	public SolenoidSet(SolenoidSubsystem system, SolenoidState state) {
+		this(system, state, () -> {
 			return false;
 		});
 	}
```

Comment:
replace with `this(system, state)`

Reply:
I'm forced to add `name` to the `this()` call because otherwise it becomes a recursive constructor invocation.